Core/include/Acts/Seeding/BinnedSPGroup.hpp

```diff
@@ -194,7 +194,7 @@ class BinnedSPGroupIterator {
   BinnedSPGroupIterator(const SpacePointGrid<external_spacepoint_t>* spgrid,
                         BinFinder<external_spacepoint_t>* botBinFinder,
                         BinFinder<external_spacepoint_t>* tBinFinder,
-                        boost::container::small_vector<size_t, 20> bins = {}) {
+                        std::vector<size_t> bins = {}) {
```

Comment:
Maybe I missed it, but why do change the type from `boost::container::small_vector` to `std::vector`?

Reply:
I changed it to `std::vector` so `zBinsCustomLooping` can be easily configured in `itk_seeding.py`. Also, I don't think we need `boost::container::small_vector` since it won't change after being initialized.

Reply:
Yeah no strong need for small vector is it's not modified.